### PR TITLE
feat: add installer script

### PR DESCRIPTION
**Pull request by sinabakh**
It would be better to open an issue and discuss it first but since it was faster to implement it I wanted to propose the change with its code.

What's your opinion regarding having such a script and making the installation seem more user-friendly (maybe).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +28,7 @@ Infracost shows cloud cost estimates for infrastructure-as-code projects such as
 
     Linux/macOS manual download:
     ```sh
-    os=$(uname | tr '[:upper:]' '[:lower:]') && \
-    arch=$(uname -m | tr '[:upper:]' '[:lower:]' | sed -e s/x86_64/amd64/) && \
-    curl -s -L https://github.com/infracost/infracost/releases/latest/download/infracost-$os-$arch.tar.gz | tar xz -C /tmp && \
-    sudo mv /tmp/infracost-$os-$arch /usr/local/bin/infracost
+    curl -fsSL https://raw.githubusercontent.com/infracost/infracost/master/scripts/install.sh | sh
```
Comment (alikhajeh1) [inline README.md:31]:
```suggestion
    # Downloads the CLI and puts it in `/usr/local/bin`
    curl -fsSL https://raw.githubusercontent.com/infracost/infracost/master/scripts/install.sh | sh
```

**Comment by alikhajeh1** [comment]
Whilst I'm always up for making things easier, there is something comforting about seeing what's about to run and seeing that it's only fetching one executable and putting it in `/usr/local/bin`. I added a suggestion so users know what the curl does.

This might just be me, feel free to ask in Slack to see how other users feel about this.

**Comment by sinabakh** [comment]
Your points make sense to me, if there were more complicated commands involved it would make more sense to have this script. However, I will ask in Slack to see if users would prefer it or not.

**Comment by sinabakh** [comment]
https://github.com/infracost/docs/pull/56

**Comment by alikhajeh1** [comment]
@sinabakh / @aliscott I added some echos so the script outputs this now:
```
Downloading latest release of infracost-darwin-amd64...

Moving /tmp/infracost-darwin-amd64 to /usr/local/bin/infracost (you might be asked for your password due to sudo)

Completed installing Infracost v0.8.3
```

ok if I merge this and the docs PR?

**Comment by aliscott** [comment]
LGTM

**Pull request merged**